batch/indexer_Position_Bond.py

```diff
@@ -343,6 +344,27 @@ def __sync_lock(self, db_session: Session, block_to: int):
                 )
             except ABIEventFunctionNotFound:
                 events = []
+            try:
+                lock_map: dict[str, dict[str, True]] = {}
+                for event in events:
+                    args = event["args"]
+                    account_address = args.get("accountAddress", "")
+                    lock_address = args.get("lockAddress", "")
+                    if lock_address not in lock_map:
+                        lock_map[lock_address] = {}
+                    lock_map[lock_address][account_address] = True
+                for lock_address in lock_map:
+                    for account_address in lock_map[lock_address]:
+                        value = self.__get_account_locked_token(token, lock_address, account_address)
+                        self.__sink_on_locked(
+                            db_session=db_session,
+                            token_address=to_checksum_address(token.address),
+                            lock_address=lock_address,
+                            account_address=account_address,
+                            value=value
+                        )
```

Comment:
Retrieving account address and lock address from events to fetch locked balance at the point of the batch cycle.